### PR TITLE
Update configure-email-forwarding.md

### LINKED ISSUE #13083: Broken link
Broken link in 'Related Content' section

https://learn.microsoft.com/en-us/email/configure-a-shared-mailbox?view=o365-worldwide#forward-emails-that-are-sent-to-a-shared-mailbox

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 5885a586-76b8-83ae-161b-6c9c8793b42d
* Version Independent ID: 681fe343-6272-a75a-ed00-dc7ef4355e67
* Content: [Configure email forwarding - Microsoft 365 admin](https://learn.microsoft.com/en-us/microsoft-365/admin/email/configure-email-forwarding?view=o365-worldwide)
* Content Source: [microsoft-365/admin/email/configure-email-forwarding.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/admin/email/configure-email-forwarding.md)
* Service: **microsoft-365-business**
* GitHub Login: @kwekuako
* Microsoft Alias: **kwekua**

### PR DESCRIPTION
**Pull request by msbemba**
Fixed the related article

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/13083

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/email/configure-email-forwarding.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ Before you set up email forwarding, note the following:
 ## Related content
 
 [Create a shared mailbox](../email/create-a-shared-mailbox.md) (article)\
-[Forward emails that are sent to a shared mailbox](/email/configure-a-shared-mailbox?view=o365-worldwide#forward-emails-that-are-sent-to-a-shared-mailbox) (article)\
+[Forward emails that are sent to a shared mailbox](https://learn.microsoft.com/microsoft-365/admin/email/configure-a-shared-mailbox?view=o365-worldwide#forward-emails-that-are-sent-to-a-shared-mailbox) (article)\
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/email/configure-email-forwarding.md]:
```suggestion
[Forward emails that are sent to a shared mailbox](/microsoft-365/admin/email/configure-a-shared-mailbox#forward-emails-that-are-sent-to-a-shared-mailbox) (article)\
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**